src/scenes/Event/components/HistoryTable/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import { EventType } from 'constants';
+import { Tabs, Tab, withStyles } from '@material-ui/core';
+
+import styles from './styles';
+import { ResultHistory, TransactionHistory } from '../';
+
+const { UNCONFIRMED, TOPIC, ORACLE } = EventType; // eslint-disable-line
+
+@withStyles(styles, { withTheme: true })
+@injectIntl
+@inject('store')
+@observer
+export default class HistoryTable extends Component {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    // oracles: PropTypes.array.isRequired,
+    intl: intlShape.isRequired, // eslint-disable-line react/no-typos
+  };
+  state = {
+    tabValue: 'transactionHistory',
```

Comment:
lets create a `const TAB_TYPE = ['resultHistory', 'transactionHistory']` to handle the tab values.